cdc/owner/owner.go

```diff
@@ -112,10 +117,16 @@ func (o *Owner) Tick(stdCtx context.Context, rawState orchestrator.ReactorState)
 		time.Sleep(1 * time.Second)
 		return state, nil
 	}
-	err = o.gcManager.updateGCSafePoint(ctx, state)
-	if err != nil {
+
+	// Owner should update GC safepoint before initializing changefeed, so
+	// changefeed can remove its "ticdc-creating" service GC safepoint during
+	// initializing.
+	//
+	// See more gcutil doc.
+	if err = o.updateGCSafepoint(stdCtx, state); err != nil {
```

Comment:
What if updating GC safepoint failed here? `updateGCSafepoint` doesn't guarantee the update op must succeed

Reply:
The current implementation update gc safepoint on a best-efforts basis. 
`updateGCSafepoint` retries internally, also we update `ticdc-creating-<ID>`'s TTL to 10min when we initialize changefeed in case `updateGCSafepoint` fails.